src/block/BlockTypeIds.php

```diff
@@ -737,8 +737,9 @@ private function __construct(){
 	public const BIG_DRIPLEAF_HEAD = 10707;
 	public const BIG_DRIPLEAF_STEM = 10708;
 	public const PINK_PETALS = 10709;
-
 	public const FIRST_UNUSED_BLOCK_ID = 10710;
+	public const MOSS_BLOCK = 10711;
```

Comment:
New block type IDs must not exceed the `FIRST_UNUSED_BLOCK_ID` value.

Reply:
So what am I supposed to do ?

Reply:
Shift first unused block type ID after Moss block and Moss carpet.

Reply:
```php
public const FIRST_UNUSED_BLOCK_ID = 10713;
```

Reply:
this must be correct ?

Reply:
Yes but you would be leaving a number gap between Moss block and petals.